### PR TITLE
grid done

### PR DESCRIPTION
**Pull request by kushalsourav**
grid component finished

### CODE REVIEW

**Review by aiman-ansari — APPROVED** [review]
Good code also grid are perfectly responsive.
Good to merge

File: packages/components/grid.css [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+}
+
+.grid-1-4 {
+    width: 100%;
+    height: auto;
+    display: grid;
+    grid-template-columns: auto auto auto auto;
+    grid-template-rows: auto;
+    gap: 0.625rem;
+}
```
Comment (anshulraheja) [inline packages/components/grid.css]: It is better to create a class for common properties in that way you do not have to rewrite the code 
Ex: you could have created a class named grid
. grid {
     width: 100%;
    height: auto;
    display: grid;
}

File: packages/components/grid.css [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+@media (max-width:500px) {
+    .grid-1-2 {
+        width: 100%;
+        height: auto;
+        display: grid;
+        grid-template-columns: auto;
+        grid-template-rows: 1fr 1fr;
+        gap: 2rem;
```
Comment (anshulraheja) [inline packages/components/grid.css]: When applying media queries, we don't need to redeclare those properties that are not changing.
So you remove width, height display, gap

**Review by anshulraheja — APPROVED** [review]

File: packages/components/grid.css (hunk 1 above)
Comment (kushalsourav) [reply packages/components/grid.css]: thanks mate

Comment (kushalsourav) [reply packages/components/grid.css]: using one class name is good right

**Review by mayuresh-tech — COMMENTED** [review]
Looks good to merge!

**Pull request merged**